ui/src/components/TransferDialog.tsx

```diff
@@ -139,43 +138,43 @@ export default function TransferDialog({ ...props }) {
         >
           <CircularProgress color="info" />
         </Backdrop>
-        <DialogContentText>
-          SSH must be enabled and configured between the source and destination
-          Docker hosts.
-        </DialogContentText>
-
-        <Grid container direction="column" spacing={2} mt={2}>
-          <Grid
-            container
-            direction="row"
-            alignItems="flex-end"
-            textAlign="center"
-            justifyContent="center"
-            spacing={1}
-          >
-            <Grid item>
-              <Button variant="outlined" disabled>
-                ssh://
-              </Button>
-            </Grid>
-            <Grid item>
-              <TextField
-                required
-                autoFocus
-                margin="dense"
-                id="dest-host"
-                label="Destination host"
-                fullWidth
-                variant="standard"
-                placeholder={"192.168.1.50"}
-                spellCheck={false}
-                onChange={(e) => {
-                  setDestHost(e.target.value);
-                }}
-              />
-            </Grid>
-          </Grid>
-          <Grid item>
+        <Alert
+          sx={(theme) => ({ marginBottom: theme.spacing(2) })}
+          severity="warning"
+        >
+          Any existing data inside the destination volume will be replaced.
+        </Alert>
+        <FormControl>
+          <FormLabel id="from-label">
+            <Typography variant="h3" my={1}>
+              From:
+            </Typography>
+          </FormLabel>
+          <VolumeOrInput />
+        </FormControl>
+        <FormControl sx={{ width: "100%" }}>
+          <FormLabel id="to-label">
+            <Typography variant="h3" mt={3} mb={1}>
+              To:
+            </Typography>
+          </FormLabel>
+          <Stack mt={1} spacing={3}>
+            <TextField
+              fullWidth
+              autoFocus
+              id="dest-host"
+              label="Destination host"
+              helperText="SSH must be enabled and configured between the source and destination Docker hosts"
```

Comment:
nit: shall we be more explicit? e.g:

"If you're having trouble retrieving the list of volumes from your remote Docker host, check that you've added the ssh public key in the `known_hosts` file."

Reply:
I kept the first sentence and added the second part of your suggestion, wdyt?

`SSH must be enabled and configured between the source and destination Docker hosts. Check you have the remote host SSH public key in your known_hosts file.`

Reply:
Perfect!